.github/workflows/code_generator.yml

```diff
@@ -58,11 +58,11 @@ jobs:
 
       - name: Check DartApi
         run: |
-          git clone --depth 1 --filter=blob:none --sparse https://github.com/dart-lang/sdk.git
+          git clone --depth 1 --filter=blob:none --sparse --branch stable https://github.com/dart-lang/sdk.git
           cd sdk
           git sparse-checkout set runtime/include
           cd ..
-          cp -rf ./sdk/runtime/include/* ./frb_rust/src/dart_api_dl/
+          cp -rf ./sdk/runtime/include/* ./frb_rust/src/dart_api/
```

Comment:
uber-nit: curious why rename dart_api_dl to dart_api? And the title suggests that we use dart-sys, so shall we remove the whole dart_api?

Reply:
renamed because only `dart_api.h` is used.
It is used to generate 
https://github.com/fzyzcjy/flutter_rust_bridge/blob/master/frb_dart/lib/src/ffi/dart_native_api.dart

Unfortunately, there is still a dependence on the `.h`. Getting rid of this addiction is not included in the PR